### ISSUE TITLE
[DOCS] Document how SDK properties affect accessibility in design system

### ISSUE THREAD

**Issue by aaronreed708** [finos/a11y-theme-builder#751]
## Problem/Concern

The goal of this issue is to write documentation around how shades, chroma, and other SDK properties affect the resulting accessibility of a design system.  This issue has the further goal of Identify what, if any, of this new documentation should be available to the user through the UI.

Related to #750

**Comment by vinfinity7** [comment]
@aaronreed708 Mind if I join this conversation and help sorting this issue out under your inputs ?

**Comment by aaronreed708** [comment]
Hi @vinfinity7, sorry for the delay in replying.  That would be wonderful if you could take a stab at this.  If you would like, we could set up a call with yourself, myself and @lwnoble to collaborate on what information should be covered and where in the documentation it should live.  And what of this information we could re-use in contextual help inside Theme Builder.

**Comment by vinfinity7** [comment]
Yes I would Love to be a part of such a call , lets make it happen . We can set up this call after 12:30 PM IST 4 May , Looking forward to work with yall !

**Comment by aaronreed708** [comment]
@vinfinity7 I can't make this weekend work.  Are you available during the week at all?  If so, what time of day is generally good for you?

**Comment by vinfinity7** [comment]
@aaronreed708 due to my university examinations , my schedule is running a bit tight nowadays ,I will be available after 12:30 PM IST 4 May onwards onto the next week , Lets make this happen !
